pkg/install/images/application_definitions.go

```diff
@@ -59,7 +59,27 @@ func GetImagesFromSystemApplicationDefinitions(
 	return images, nil
 }
 
-type SystemAppsHelmChart struct {
+func GetImagesFromDefaultApplicationDefinitions(
+	logger logrus.FieldLogger,
+	config *kubermaticv1.KubermaticConfiguration,
+	helmClient helm.Client,
+	helmTimeout time.Duration,
+	registryPrefix string,
+) ([]string, error) {
+	var images []string
+
+	for defaultApp, err := range DefaultAppsHelmCharts(config, logger, helmClient, helmTimeout, registryPrefix) {
+		if err != nil {
+			return nil, err
+		}
+
+		images = append(images, defaultApp.WorkloadImages...)
+	}
+
+	return images, nil
```

Comment:
No need for extra spacings and the indentation for the if block looks really off here. Can you please run `gofmt -s -w .`